pages/in/settings/others.vue

```diff
@@ -197,4 +314,12 @@ export default {
     }
   }
 }
+
+.flex {
+  display: flex;
```

Comment:
For align children vertically center
```suggestion
  display: flex;
  align-items: center;
```